retrieval/discovery/marathon/marathon.go

```
@@ -217,7 +220,29 @@ func fetchApps(client *http.Client, url string) (*AppList, error) {
 		return nil, err
 	}
 
-	return parseAppJSON(body)
+	// return parseAppJSON(body)
+	rawAppList, err := parseAppJSON(body)
+	if err != nil {
+		log.Error("Error parsing app JSON body: ", err)
```

we should return an error I guess. Previously, we returned it.

---

Well spotted, DONE.